### ISSUE TITLE
Cover text color on song screen is always white (ignoring skin / theme)

### ISSUE THREAD

**Issue by daggeg** [Vocaluxe/Vocaluxe#444]
There seems to be a mix up between the genius theme skins. For instance, I use the Red Crimson skin and the last nightly seem use the text color from the Standard skin to a degree (as shown in the attached pic).
![Screenshot_4](https://user-images.githubusercontent.com/6724256/72364779-58089180-36f7-11ea-8454-4973defb53b4.png)

**Comment by daggeg** [comment]
I also just noticed that when using the Ice Blue skin it gets mixed up with Crimson Red when doing Challenge Mode.

**Comment by Stefan1200-de** [comment]
I just started a nightly from 2018 (23.09.2018) and used similar settings than I can see on your screenshot, it looks the same. So in the last years nothing was changed here. Maybe you changed more stuff at the theme files, which made that white color red?
![image](https://user-images.githubusercontent.com/15168351/72367981-78d3e580-36fd-11ea-9258-8d0e6958c646.png)

**Comment by daggeg** [comment]
Hm, it's been looking the same since late 2017 at least. Perhaps I did something back then? This is what it looked like at my end prior to the last nightly. Which kind of makes sense since I'm using Crimson Red.
![Screenshot](https://user-images.githubusercontent.com/6724256/72368440-627a5980-36fe-11ea-861f-192511a59bc7.png)

**Comment by daggeg** [comment]
Stefan, I thought you changed the text color to white since you were not a fan of the genius theme well enjoying the white text color :)

**Comment by Stefan1200-de** [comment]
Please ZIP and Upload your Themes folder of your installation from 2017.

**Comment by daggeg** [comment]
> I just started a nightly from 2018 (23.09.2018) and used similar settings than I can see on your screenshot, it looks the same. So in the last years nothing was changed here. Maybe you changed more stuff at the theme files, which made that white color red?
Are you sure about that? I could use my old theme files in the first nightly you produced, but after the last one the game crashes when I copy my old 03_Crimson_Red file. This code wasn't there until recently (I reckon):
   <TextPauseColor>
      <R>1.000</R>
      <G>1.000</G>
      <B>1.000</B>
      <A>0.900</A>
    </TextPauseColor>

**Comment by Stefan1200-de** [comment]
> Stefan, I thought you changed the text color to white since you were not a fan of the genius theme well enjoying the white text color :)

😂

No, I would just add a new xml file with my preferred colors ;)

**Comment by Stefan1200-de** [comment]
Just to put my knowledge together at this place:
- It must be a change to the nightly between mid 2017 and mid 2018, because mid 2018 it was already the wrong color, but mid 2017 it is the right text color.
- That bug only exists in the TileBoard view, in the list view all colors are correct.
- It seems that the issue is not XML file related (Genius Theme files), because they are almost unchanged (with the exception of new added stuff, which should not be the reason for it).

**Comment by Stefan1200-de** [comment]
Found the line, the color is hard coded:
https://github.com/Vocaluxe/Vocaluxe/blob/develop/VocaluxeLib/Menu/SongMenu/CSongMenuTileBoard.cs#L588

But I don't see the change of this line in the file history?!? Why was this right in the mid 2017 nightly?

**Comment by Stefan1200-de** [comment]
That particular line was last changed 7 years ago:
![image](https://user-images.githubusercontent.com/15168351/72372924-9d34bf80-3707-11ea-8ee8-fe24025f1b32.png)

**Comment by daggeg** [comment]
I noticed that my older nightly, where this problem didn't exist, the genius crimson red skin make use of data from the ambient theme. For instance, these folder graphics from the ambient red skin are used by the genius crimson red skin.
![image](https://user-images.githubusercontent.com/6724256/72377172-8abe8400-370f-11ea-953b-5883b84dbfc9.png)

**Comment by daggeg** [comment]
@Jiiniasu, do you know what this is all about?

**Comment by Stefan1200-de** [comment]
I have to change one thing I wrote 2 hours ago (I already edited my old post). Even in the list view is a hard coded color, but it is a little bit different.
![image](https://user-images.githubusercontent.com/15168351/72380950-cd378f00-3716-11ea-8e3b-a04d2cedc921.png)

**Comment by daggeg** [comment]
And I just found out about this is my old config file. Perhaps it was me myself who had at least guided the program to use the ambient theme graphics?
![image](https://user-images.githubusercontent.com/6724256/72381811-51d6dd00-3718-11ea-99e3-a83eb4799105.png)

**Comment by Stefan1200-de** [comment]
I was able to switch that color to the selected skin color, but it looks bad in my opinion. I guess it was changed by intention to hard coded white, because it looks best. But see here:
![image](https://user-images.githubusercontent.com/15168351/72382649-032a4280-371a-11ea-8c0e-11f2120fd3f9.png)

**Comment by daggeg** [comment]
Ok. I just copied my old ambient red cover folder and changed the config to have the skin name Crimson red.
![image](https://user-images.githubusercontent.com/6724256/72383209-4507b880-371b-11ea-81d8-317dc5d9911d.png)
And now the last nightly gives me the following:
![Screenshot](https://user-images.githubusercontent.com/6724256/72383251-5ea90000-371b-11ea-9895-7e053443d336.png)

**Comment by daggeg** [comment]
So now I'm back where I want to be :D

**Comment by Stefan1200-de** [comment]
I don't understand, what did you to get there?

**Comment by daggeg** [comment]
I copied the old cover folders for the ambient theme to the present cover folder like this:

![image](https://user-images.githubusercontent.com/6724256/72383777-7339c800-371c-11ea-8578-57b4c6d956b4.png)

Then I edited the active config file in the following way:
  <Theme>
    <Name>Genius</Name>
    
    <Skin>Crimson Red</Skin>
    
    **<Cover>Ambient Red</Cover>**
Somehow that seems to do the trick.

**Comment by Stefan1200-de** [comment]
Ah, that was important for me to understand where that colors are defined. So it's quite easy, I just created a **Genius Red.xml** and **Genius Blue.xml** file for the Cover directory, no need for other additions. Maybe I should add them for testing. I found also a small bug, if you switch that file using the game options, without restarting the game, will corrupt the display of the song list. But after a Vocaluxe restart everything is fine.
